### ISSUE TITLE
Feature: selectable keys for replace command

### ISSUE THREAD

**Issue by mattlqx** [fishi0x01/vsh#72]
While the `replace` command will be a powerful way to make changes to a Vault store, it may be difficult to limit replacements in cases where you only want to change a particular key across many paths.

For example, if you have the following Vault structure:

```
my/vault/path/a:
  first-key: weak-value
  second-key: strong-value
...
my/vault/path/z:
  first-key: strong-value
  second-key: weak-value
```

And you only want to change `first-key`'s value from `weak-value` to `strong-value`, you can't presently do that with the replace command without also changing `second-key`'s value. This is a contrived example, but most of the time, you will indeed want to limit changing values to a particular key.

**Comment by fishi0x01** [comment]
I think the use-case you describe is quite common. That feature proposal is a great idea. I will give it a try on the weekend :+1:

**Comment by mattlqx** [comment]
I guess I'm trying to inform you what I'm up to by opening these. I already have it done, but was dependent on the prior PR. 🙂  I also did notice some bugs with regex matching that I'll fix soon too as well.

After that though, I think I can take a break. lol

**Comment by fishi0x01** [comment]
Haha ok. Well in that case, thanks again Matt. That is great news :bow:

**Comment by fishi0x01** [comment]
Fixed by #74
